File: 백준/Silver/1475. 방 번호/방 번호.py
import sys
input = sys.stdin.readline

s = list(map(int, input().strip()))

num = [0 for _ in range(9)]
num[6] += 1
for i in s :
    if i == 9 :
        num[6] += 1
    else :
        num[i] += 1
num[6] = num[6] // 2

print(max(num))

File: 백준/Silver/1475. 방 번호/방 번호.java
import java.io.*;
import java.util.*;
public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int N = Integer.parseInt(br.readLine());
		int[] num = new int[10];
		while (N>0) {
			num[N%10]++;
			N/=10;
		}
		num[6] = (num[6] + num[9] + 1)/2;

        int max = 0;
		for (int i = 0; i < 9; i++) {
			max = Math.max(max, num[i]);
		}
		System.out.println(max);
	}
}